Src/ILGPU/IntrinsicMath.cs

```diff
@@ -683,7 +683,7 @@ public IntegerParts(uint lower, uint upper)
         [MathIntrinsic(MathIntrinsicKind.CopySignF)]
         public static float CopySign(float x, float y) =>
 #if !NETFRAMEWORK && !NETSTANDARD
-            Math.CopySign(x, y)
+            (float)Math.CopySign(x, y);
```

Comment:
ISSUE: This should be `MathF`.

I noticed it the other day, but didn't think there was a rush to fix it 😇

Reply:
Thanks for spotting this @MoFtZ!

Fixed in the next push.